make/data/charsetmapping/charsets

```diff
@@ -146,7 +146,7 @@ charset ISO-8859-1 ISO_8859_1
 charset ISO-8859-2 ISO_8859_2
     package sun.nio.cs
     type    sbcs
-    hisname ISO8859_2
+    histname ISO8859_2
```

Comment:
Should this column be re-aligned with the longer name?

Reply:
I thought about it before publishing the PR. I decided not to re-align anything because (i) the change would be bigger and (ii) the fact that there was already a property that is similarly misaligned; search for:

    internal true

Reply:
If you are concerned with functionality rather than looks, then I can tell you this:

1. The build succeeds and tier1 tests pass.
2. The code that parses that file expect one or more whitespace characters as a separator:

        String[] tokens = line.split("\\s+");